SBOM_LIFECYCLE.md

```diff
@@ -0,0 +1,42 @@
+# SBOM Life Cycle
+
+![https://raw.githubusercontent.com/CISA-SBOM-Community/SBOM-Generation/main/assets/lifecycle.svg]
+
+# Generation
+The generation incorporates the steps required to generate an NTIA Minimum Elements and/or Framing Document SBOM. The life cycle is format agnostic and is the same for both SPDX and CycloneDX (even though the tools may differ).
+
+## Creation
+The creation step is where SBOM is conceived. The level of "completeness" of this SBOM will vary a great amount depending on the tool used, as well as what language you're working with. It will also matter if the SBOM is a source or build SBOM. Using build SBOMs is recommended to achieve higher completeness.
```

Comment:
Do we define "source SBOM" and "build SBOM" anywhere?

Reply:
Yes, this is referenced in the main README.